tests/device_tests/test_msg_signtx_decred.py

```diff
@@ -44,6 +44,14 @@
 TXHASH_16da18 = bytes.fromhex(
     "16da185052740d85a630e79c140558215b64e26c500212b90e16b55d13ca06a8"
 )
+TXHASH_8b6890 = bytes.fromhex(
+    "8b6890c10a3764fe6f378bc5b7e438148df176e9be1dde704ce866361149e254"
+)
+TXHASH_1f00fc = bytes.fromhex(
+    "1f00fc54530d7c4877f5032e91b6c507f6a1531861dede2ab134e5c0b5dfe8c8"
+)
+
+pytestmark = [pytest.mark.skip_t1]
```

Comment:
this should not be here...

Reply:
Oh yeah, my bad.

Reply:
https://github.com/trezor/trezor-firmware/pull/1249/commits/8b972ccf3344c97fd616e0d0c89b3101a8772c2b